src/test/java/dk/kb/cfutv/utils/RitzauHarvestUtilTest.java

```diff
@@ -58,13 +60,13 @@ public void smallIntervalTest() throws ParseException {
     
     @Test
     public void slightlyBiggerIntervalTest() throws ParseException {
```

Comment:
ParseException skal vist helt væk, for ZoneDateTime smider DateTimeParseException. Det gælder alle efterfølgende metoder

Reply:
De skulle gerne være rettet til DateTimeParseExceptions (Idet ZDTs vel også smider dem hvis de parser noget som ikke er på format)